libcxx/test/std/thread/thread.mutex/thread.lock/thread.lock.guard/types.pass.cpp

```diff
@@ -23,10 +21,10 @@
 
 #include "test_macros.h"
 
-int main(int, char**)
-{
-    static_assert((std::is_same<std::lock_guard<std::mutex>::mutex_type,
-                   std::mutex>::value), "");
+struct MyMutex {};
+
+int main(int, char**) {
```

Comment:
No `main` needed, and this can be a `.compile.pass.cpp`

Reply:
Done.